server/backend.py

```diff
@@ -1,14 +1,70 @@
 from flask import Flask, request, jsonify
+import json
+import pickle
+import pandas as pd
+
 
 app = Flask(__name__)
 
 # Your ML model for detecting dark patterns
+
+def load_models():
+    rfc = 'Dark-Patterns-Buster-v2\model\rfc.pkl'
+    svm = 'Dark-Patterns-Buster-v2\model\svc.pkl'
+    vectorizer = 'Dark-Patterns-Buster-v2\model\tfidf_vectorizer.pkl'
+    df = pd.read_csv('Dark-Patterns-Buster-v2\model\dataset.tsv', sep="\t", encoding="utf-8")
```

Comment:
Still reading the complete dataset for each api call?

Reply:
Just to create the map, either we can hardcode the map.

Reply:
Let me hardcode the map and assign model files to global variables.